presto-native-execution/presto_cpp/presto_protocol/Connectors.cpp

```diff
@@ -44,6 +44,10 @@ void registerHiveConnectors() {
   registerConnector("hive-hadoop2", "hive");
 }
 
+void registerIcebergConnector() {
+  registerConnector("iceberg", "iceberg");
```

Comment:
We need a different connectorKey(second argument in registerConnector) for iceberg. This is required because Iceberg Connector's Plan Fragment is different from Hive. 

Since registerHiveConnectors had connector's with connectorKey as `hive`, I decided to include this in a new method. But if its fine to include `registerConnector("iceberg", "hive-iceberg");` inside registerHiveConnectors, we can do that.

Reply:
@imjalpreet I prefer we merge the content of registerIcebergConnector() to registerHiveConnectors() and remove the registerIcebergConnector(). This is the connectors for Velox, which doesn't have Iceberg connector.

```
void registerHiveConnectors() {
  registerConnector("hive", "hive");
  registerConnector("hive-hadoop2", "hive");
  registerConnector("iceberg", "hive-iceberg");
}
```
@majetideepak @aditi-pandit @Yuhta which way do you prefer?

Reply:
Sure, I will do that.

Reply:
Agree that keep it with hive is preferable as long as it does not introduce new third party dependency